codegen/src/main/java/org/nd4j/codegen/impl/java/Nd4jNamespaceGenerator.java

```
@@ -70,19 +74,49 @@
 
     private Nd4jNamespaceGenerator() { }
 
-    public static void generate(NamespaceOps namespace, GeneratorConfig config, File outputDirectory, String className) throws IOException {
-        String basePackage = "org.nd4j.linalg.factory";
+    public static void generate(NamespaceOps namespace, GeneratorConfig config, File outputDirectory, String className,
+                                String basePackage) throws IOException {
+        //String basePackage = "org.nd4j.linalg.factory";
 
         generateEnums(outputDirectory, basePackage);
         generateConfigs(outputDirectory, basePackage);
-        generateOpFactory(namespace, outputDirectory, className, basePackage);
+        try {
+            generateOpFactory(namespace, outputDirectory, className, basePackage, StringUtils.EMPTY);
+        }
+        catch (Exception e) {
+
+        }
```

Don't silently swallow exception here

---

Fixed.